src/utils/helpers.js

```diff
@@ -58,6 +58,22 @@ export function parseJSON(response) {
     })
   }
 
+  if (response.status === 429) {
+    // Only the body of the response is passed back so to catch a 429
+    // we fill the JSON body with some error details to give information
+    // to the consumer
+    const error = {
+      errors: [{ status: 429 }]
```

Comment:
@craigtweedy Not want to return a `title` or `detail`/`message` here? `429` is quite ambiguous.

Reply:
Why do you feel 429 is ambiguous? Surely it only means the one thing?

I thought about a title and detail here - but didn't want to start the task of properly defining our own error codes and messages.

Reply:
Then again, the status may not always be the same as the HTTP status code, and to be clearer for devs that may not know that's a rate limit it is probably best to be clear.